server.js

```diff
@@ -3386,7 +3385,7 @@ cache(function(data, match, sendBadge, request) {
     try {
       var data = JSON.parse(buffer);
       var issues = data.total_count;
-      var rightText = isRaw? '': (isClosed? ' closed': ' open');
+      var rightText = isClosed? ' closed': (isRaw? '': ' open';
```

Comment:
Missing brace here. If you fix it quick, we might bring this to the server quite fast!

Reply:
Right, should be fixed now. I also reorganized the tests, because there were some duplicates (after rebasing).